### PR TITLE
Add naUSD to kujira

### PR DESCRIPTION
**Pull request by jpKuji**
This PR aims to add the receipt token of NAMI Protocol to the registry.

### CODE REVIEW

File: kujira/assetlist.json [hunk 1]
```diff
@@ -2463,6 +2463,33 @@
...
+      "display": "naUSD",
+      "symbol": "NAUSD",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nausd.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nausd.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nausd.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/nausd.svg"
```
Comment (JeremyParish69) [inline kujira/assetlist.json:2489]: The images within the same image object must be visually identical, but the PNG "N" is sans-serif, which the SVG "N" has serif.

File: kujira/assetlist.json [hunk 2]
```diff
@@ -2463,6 +2463,33 @@
...
+          "exponent": 0
+        },
+        {
+          "denom": "nausd",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/kujira18thffdpuz8tm3mwwun4uhks8a2v7ye0jmw3fllqhklma82lqkpms20gh9c/unausd",
+      "name": "NAUSD",
+      "display": "naUSD",
```
Comment (JeremyParish69) [inline kujira/assetlist.json:2480]: this display denom is not defined (units are case sensitive). "nausd" is a defined denom, though.

File: kujira/assetlist.json (hunk 1 above)
Comment (jpKuji) [reply kujira/assetlist.json:2489]: Hey, thanks. Unfortunately I'm not sure I understand your comment correctly. The "N" in the svg is of type Futura, which is sans-serif.

```
<text
       xml:space="preserve"
       style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:341.333px;font-family:Avenir;-inkscape-font-specification:'Avenir Bold';fill:#1b1b1b;fill-opacity:1"
       x="100.52281"
       y="384.68256"
       id="text2"><tspan
         id="tspan2"
         x="100.52281"
         y="384.68256"
         style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:341.333px;font-family:Futura;-inkscape-font-specification:'Futura Bold';fill:#d3fffe;fill-opacity:1">N</tspan></text>
```

Basically I created the SVG in inkscape and just exported it as a png.

Comment (JeremyParish69) [reply kujira/assetlist.json:2489]: The "N" renders weirdly in the SVG.
You can see how they don't match:
image

Comment (jpKuji) [reply kujira/assetlist.json:2489]: Needed to convert the text to a path. Should be okay now.

**Pull request merged**